keps/sig-scheduling/3902-decoupled-taint-manager/README.md

```diff
@@ -173,14 +178,18 @@ Since there are no new API changes, we can skip Alpha and go to Beta directly.
 
 #### GA
```

Comment:
do we have feedback on this extensibility was actually used?

Reply:
This feature gate is enabled by default when it is introduced since 1.29. I don't know how to prove it is being used. Can you help me with this? Thanks.

/cc @yuanchen8911 @atosatto

Reply:
the feature has a clear motivation to enable new use cases. If nobody tried those new use cases, we cannot tell that this feature is ready for stable. Any pointers on how it was used would be great to have as an evidence

Reply:
this is still not addressed, correct? It will be great to demonstrate the use of this feature.

Reply:
I tried to contact him via Slack yesterday, but have not received a response yet.

Reply:
We have some usecases where we don't want the workloads to be deleted(stateful workloads), we use this featuregate at Apple to disable taintmanager so that we don't need to maintain a webhook for the delete operations on the pod.

Reply:
I actually prefer to leave the "extensibility" use cases out.

See the motivation and non-goals:
https://github.com/kubernetes/enhancements/blob/4b8baf578b0cf7ae1733528dd2cd8a79a30fca2e/keps/sig-scheduling/3902-decoupled-taint-manager/README.md#non-goals

I would encourage whoever disables the controller to rather contribute upstream to enhance the API to be able to express the needs that you have. Simply disabling the controller and implement your own might make that particular cluster non-conformant.